src/components/EmojiSuggestions/index.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import BaseEmojiSuggestions from './BaseEmojiSuggestions';
+import * as DeviceCapabilities from '../../libs/DeviceCapabilities';
+import {propTypes, defaultProps} from './emojiSuggestionsPropTypes';
+
+const EmojiSuggestions = (props) => {
+    const containerRef = React.useRef(null);
+    React.useEffect(() => {
+        const container = containerRef.current;
+        if (!container) {
+            return;
+        }
+        container.onpointerdown = (e) => {
```

Comment:
Please add code comments explaining _why_ this logic is here.

Reply:
OK, sure, I'll expand my suggestion further. When I look at this code without having any context, my mind immediately asks these questions:

1. Why is this a platform specific file?
2. What is it's purpose?
3. Why is there a handler for onpointerdown?
4. Why is that event being handled?

A code comment should hopefully explain most of those things for the next engineer that sees the code. It could be something like this:

> mWeb chrome browsers behave differently with long-press events. The consistent behavior that is desired on all platforms is that long-press on the emoji suggestion will do nothing. In order to achieve this consistency on mWeb chrome, the onpointerdown event needs to be ignored. If it's not ignored, then a focus event is triggered and the tapped on emoji is added to the composer which isn't what we want.

Reply:
Oh, I see you added a bit of a comment above (sorry I didn't check the code first). I'll resolve this thread